resolve/LinSolverIterative.hpp

```diff
@@ -43,7 +43,7 @@ namespace ReSolve
       index_type total_iters_;
 
       // Parameters common for all iterative solvers
-      real_type tol_{1e-14};  ///< Solver tolerance
+      real_type tol_{constants::DEFAULT_TOL};  ///< Solver tolerance
```

Comment:
I suggest something that is easier to understand:
```suggestion
      real_type tol_{10*constants::MACHINE_EPSILON};  ///< Solver tolerance
```